### PR TITLE
Align table headers 💅

### PR DESCRIPTION
**Pull request by pierreericgarcia**

### CODE REVIEW

**Comment by pierreericgarcia** [comment]
[ETQ U, je vois les titres des colonnes de la liste des habilitations alignés](https://trello.com/c/LatTcCHa/1664-etq-u-je-vois-les-titres-des-colonnes-de-la-liste-des-habilitations-align%C3%A9s)

File: frontend/src/components/atoms/inputs/Input.js [hunk 1]
```diff
@@ -20,30 +20,38 @@ export const Input = ({
...
+      id={id}
+      readOnly={disabled}
+      value={value}
+      aria-label={ariaLabel}
+      required={required}
+      {...rest}
+    />
   );
+
+  if (label) {
```
Comment (rdubigny) [inline frontend/src/components/atoms/inputs/Input.js:39]: The usage of input without label is forbidden by the dsfr and make the input not accessible. This usage should not be encouraged in my opinion. I'd rather use the <input> react element directly in the table header 🤔

**Review by rdubigny — CHANGES_REQUESTED** [review]
checkboxes are not align with labels anymore:

image

there is too much space here:

image

File: frontend/src/components/organisms/Table/style.css [hunk 2]
```diff
@@ -46,12 +46,35 @@
...
+
+.datapass-checkbox-filter.fr-checkbox-group input[type=checkbox]+label {
+  padding: 0.4rem 0;
+}
+
+.datapass-checkbox-filter.fr-checkbox-group input[type=checkbox]+label:before {
+  margin: 0.4rem 0;
+}
+
+.fr-label + .fr-input, .fr-label + .fr-input-wrap, .fr-label + .fr-select {
```
Comment (rdubigny) [inline frontend/src/components/organisms/Table/style.css:64]: this rule is not selective enought and can have unexpected effect on the app. Plz scope this under .datapass-table selector.

**Review by rdubigny — CHANGES_REQUESTED** [review]

**Review by rdubigny — APPROVED** [review]

**Pull request merged**